generated/nidcpower/nidcpower_service.cpp

```diff
@@ -30,13 +33,6 @@ namespace nidcpower_grpc {
   {
   }
 
-  void NiDCPowerService::Copy(const std::vector<ViBoolean>& input, google::protobuf::RepeatedField<bool>* output) 
```

Comment:
These were all identical and are moved to converters.h (with a new name: convert_from_grpc)